lisp/tree-sitter-query-faces.el

```diff
@@ -0,0 +1,821 @@
+;; tree-sitter-query-faces.el --- faces for query matches -*- lexical-binding: t; coding: utf-8 -*-
+
+;; Copyright (C) 2019  Tuấn-Anh Nguyễn
+;;
+;; Author: Jorge Javier Araya Navarro <jorgejavieran@yahoo.com.mx>
+
+;; This file contains faces which represent different sequences of query
+;; matches when running queries in the query builder
+
+;; Code:
+
+(defface tree-sitter-query-faces-match-1
```

Comment:
IMO it'd be easier to push these through with a macro-let.

Reply:
> IMO it'd be easier to push these through with a macro-let.

is that  `cl-macrolet` by chance? I don't know anything about macro-let.

Reply:
Yes. That way you can let-bind an anonymous one-off macro without namespace pollution. Here's an example: https://github.com/Alexander-Miller/treemacs/blob/master/src/elisp/treemacs-core-utils.el#L885